db/seeds/development/6_responsibilities.rb

```diff
@@ -0,0 +1,10 @@
+def generate_responsibility()
+  new_responsibility = Responsibility.create(
+    status: "completed",
+  )
+end
+
+User.all.each do |user|
+  generate_responsibility()
+  puts "Created responsibility for #{user.first_name} #{user.last_name}."
```

Comment:
Once my pull is in, this will just be:
` puts "Created responsibility for #{user.name}."`